target/linux/kirkwood/base-files/lib/preinit/07_set_iface_mac

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+preinit_set_mac_address() {
+	local mac
+
+	. /lib/functions.sh
```

Comment:
I guess you copied this file from `target/linux/mediatek/mt7623/base-files/lib/preinit/07_set_iface_mac`. But I don't think there's a need to source files inside a function here. See `target/linux/ath79/nand/base-files/lib/preinit/10_fix_eth_mac.sh`.
BTW do we actually need it? It seems `package/base-files/files/etc/preinit` already covers it.